api/src/neo4j/queries/externalDb.js

```diff
@@ -1,10 +1,22 @@
 import querySingleResult from 'neo4j/queryHandlers/single';
 
 const getComponentsForExternalDb = async ({ dbName, externalId }) => {
-  const statement = `
+  let statement = `
 MATCH (db:ExternalDb {dbName: '${dbName}', externalId: '${externalId}'})-[v]-(c)
 RETURN { externalDb: properties(db), components: COLLECT({ component: c, version: type(v) }) }
 `;
+  if (dbName === 'MetabolicAtlas') {
+    statement = `
+MATCH (r:Reaction {id: '${externalId}'})-[v]-(c)
```

Comment:
It doesn't look like the other node's details are relevant here:
```suggestion
MATCH (r:Reaction {id: '${externalId}'})-[v]-()
```

Reply:
Sorry that I missed this scenario. It has been fixed by 22495836.
The refactoring of the neo4j querying has been done by 037371d0 and  4c33332d